audit/projects/kubernetes-public/iam.json

```diff
@@ -123,7 +123,7 @@
     },
     {
       "members": [
-        "serviceAccount:service-127754664067@serverless-robot-prod.iam.gserviceaccount.com"
+        "deleted:serviceAccount:service-127754664067@serverless-robot-prod.iam.gserviceaccount.com?uid=118182660088477675409"
```

Comment:
not entirely sure what this is about... 

/cc @listx @tylerferrara 
would this be from shutting down the auditor?

Reply:
I don't think this has anything to do with the cip auditor that runs in Cloud Run.

Reply:
@spiffxp I accidentally enabled Cloud Run API on the project. Sorry for the confusion. 🙇🏾